### PR TITLE
Removed redundant text in doc

### PR DESCRIPTION
**Pull request by yqszxx**
Removed redundant text in doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ This is in contrast to the following hardware oriented testers, in which a testi
 circuit, or device under test (DUT) from value vectors for each input.  Testing the outputs each cycle against a separate
 set of value vectors for each output.
 
-For a longer descripton see the [Using the PeekPokeTester](https://github.com/ucb-bar/chisel-testers/wiki/Using%20the%20PeekPokeTester) see the 
+For a longer descripton see the [Using the PeekPokeTester](https://github.com/ucb-bar/chisel-testers/wiki/Using%20the%20PeekPokeTester)
```
Comment (edwardcwang) [inline README.md]:
```suggestion
For a longer description, see the [Using the PeekPokeTester](https://github.com/ucb-bar/chisel-testers/wiki/Using%20the%20PeekPokeTester).
```

Comment (edwardcwang) [inline README.md]:
```suggestion
For a longer description, see the [Using the PeekPokeTester](https://github.com/ucb-bar/chisel-testers/wiki/Using%20the%20PeekPokeTester).
```

File: README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ This is in contrast to the following hardware oriented testers, in which a testi
 circuit, or device under test (DUT) from value vectors for each input.  Testing the outputs each cycle against a separate
 set of value vectors for each output.
 
-For a longer descripton see the [Using the PeekPokeTester](https://github.com/ucb-bar/chisel-testers/wiki/Using%20the%20PeekPokeTester) see the 
+For a longer description, see the [Using the PeekPokeTester](https://github.com/ucb-bar/chisel-testers/wiki/Using%20the%20PeekPokeTester).
```
Comment (edwardcwang) [inline README.md]:
```suggestion
For a longer description, see the [Using the PeekPokeTester](https://github.com/freechipsproject/chisel-testers/wiki/Using%20the%20PeekPokeTester).
```

Comment (yqszxx) [reply README.md]: Yep totally agree^_^

**Comment by yqszxx** [comment]
The old link redirects to this repository anyway🤔

**Comment by edwardcwang** [comment]
We may as well keep it up to date if we're going to update the line.

**Comment by edwardcwang** [comment]
We may as well keep it up to date if we're going to update the line.

**Pull request merged**